### PR TITLE
doc: reorganized private locations documentation tree, closes HYB-168

### PR DESCRIPTION
**Pull request by tpetillot**
[doc: reword private locations configuration page](https://github.com/gatling/frontline-cloud-doc/commit/64192a5531d27e6b29366e93a63f83f9947c38db)

[doc: document control plane token with environment variable and system property](https://github.com/gatling/frontline-cloud-doc/commit/972807a61b8853af421aa1668ecd89f9f713ac2d) 

[doc: reorganized private locations documentation tree, closes HYB-168](https://github.com/gatling/frontline-cloud-doc/commit/de103c93d700ddf163c76bb35ed17f9a5c9a7446) 

**Modifications:**

Private locations documentation tree is now:
* Introduction
* Configuration
  * AWS EC2
  * Kubernetes
* Deployment
  * ECS
  * Kubernetes

### CODE REVIEW

**Review by slandelle — CHANGES_REQUESTED** [review]
* `Deployment` is too vague for me. Installation?
* Shouldn't Installation go BEFORE Configuration?
* Isn't there a common configuration bloc, in particular the token? Shouldn't we have a dedicated section?
* AWS EC2 (too vague) => AWS EC2 Load Generators
* Kubernetes (too vague) => Kubernetes Load Generators

File: content/reference/admin/private_locations/_index.md [hunk 1]
```diff
@@ -1,11 +1,9 @@
...
-weight: 22050
+weight: 22030
 ordering:
+  - introduction
   - configuration
-  - ecs_deployment
-  - ec2_configuration
-  - kubernetes_deployment
-  - kubernetes_configuration
+  - deployment
```
Comment (slandelle) [inline content/reference/admin/private_locations/_index.md:8]: forgot to rename here?

**Review by slandelle — CHANGES_REQUESTED** [review]

File: content/reference/admin/private_locations/configuration/kubernetes/index.md [hunk 2]
```diff
@@ -24,6 +24,10 @@ When connecting to the cluster using HTTPS, if a custom truststore and/or keysto
 control-plane {
   # Control plane token
   token = "cpt_example_c7oze5djp3u14a5xqjanh..."
+  # Control plane token with an environment variable
+  token = ${?CONTROL_PLANE_TOKEN}
+  # Control plane token with a system property
+  token = $?control.plane.token
```
Comment (bastien-gatling) [inline content/reference/admin/private_locations/configuration/kubernetes/index.md:30]: Same remark : i would comment the two other examples.

File: content/reference/admin/private_locations/introduction/index.md [hunk 3]
```diff
@@ -60,24 +60,51 @@ The status has three possible values:
...
-### Locations
+**Configuration include:**
+```bash
+control-plane {
+  # Control plane token
+  token = "cpt_example_c7oze5djp3u14a5xqjanh..."
+  # Control plane token with an environment variable
+  token = ${?CONTROL_PLANE_TOKEN}
+  # Control plane token with a system property
+  token = $?control.plane.token
```
[the remark this answers is not in the document]
Comment (tpetillot) [reply content/reference/admin/private_locations/introduction/index.md:76]: as it's an optional override, not sure if we should comment them.

**Review by bastien-gatling — APPROVED** [review]

**Pull request merged**